File: Python/Product.py
class Product:

    # Product attributes.
    __price = 0
    __idProduct = ""

    # Constructor.
    # Please take note that after a few try, Python seems to not accept
    # multiple constructor. So I decided to set it as empty constructor. #

    # Empty constructor.
    def __init__(self):
        pass
    
    # Setter and Getter. #

    # Set price attribute.
    def setPrice(self, price):
        self.__price = price
    
    # Set product's ID attribute.
    def setIDProduct(self, idProduct):
        self.__idProduct = idProduct
    
    # Get price attribute.
    def getPrice(self):
        return self.__price
    
    # Get product's ID attribute.
    def getIDProduct(self):
        return self.__idProduct

File: PHP/Product.php
<?php
    // Class Product, treated as superclass.
    class Product
    {
        // Product attributes.
        private $price;
        private $idProduct;

        /* Constructor.
           Please take note that after a few try, PHP seems to not accept
           multiple constructor. So I decided to set it as empty constructor. */
        
        // Empty constructor.
        public function __construct()
        {
            
        }

        /* Setter and Getter. */

        // Set price attribute.
        public function setPrice($price)
        {
            $this->price = $price;
        }

        // Set product's ID attribute.
        public function setIDProduct($idProduct)
        {
            $this->idProduct = $idProduct;
        }

        // Get price attribute.
        public function getPrice()
        {
            return $this->price;
        }

        // Get product's ID attribute.
        public function getIDProduct()
        {
            return $this->idProduct;
        }

        /* Destructor. */
        
        public function __destruct()
        {
            
        }
    }
?>